### PR TITLE
MAUI-2361 Added UG documentation for Slider edge labels placement support

### PR DESCRIPTION
**Pull request by PraveenGopalsamy**
Hi Team,

I have added Ug documentation for the Sliders(SfSlider, SfRangeSlider, and SfRangeSelector) edge label placement feature. Please review the content and merge this PR.

Regards,
Praveen G.

### CODE REVIEW

**Review by MeikandaNayanar — APPROVED** [review]

**Review by MeikandaNayanar — APPROVED** [review]

**Comment by PraveenGopalsamy** [comment]
rebuild

**Comment by PraveenGopalsamy** [comment]
rebuild

**Comment by PraveenGopalsamy** [comment]
rebuild

File: MAUI/Range-Selector/labels.md [hunk 1]
```diff
@@ -218,6 +218,63 @@ rangeSelector.Content = chart;
 
 N> Refer [here](https://help.syncfusion.com/maui/range-slider/events) to customize label text format through range slider events.
 
+## Edge labels placement
+
+The `EdgeLabelsPlacement` property is used to move the first and last label either inside the track bounds or on the intervals. The default value of the `EdgeLabelsPlacement` property is `SliderEdgeLabelPlacement.Default`.
```
Comment (MeikandaNayanar) [inline MAUI/Range-Selector/labels.md:223]: based on the intervals

**Review by MeikandaNayanar — CHANGES_REQUESTED** [review]

File: MAUI/Range-Selector/labels.md (hunk 1 above)
Comment (PraveenGopalsamy) [reply MAUI/Range-Selector/labels.md]: Updated as mentioned.

**Review by MeikandaNayanar — APPROVED** [review]

**Pull request merged**